plexmuxy_gui/app.py

```
@@ -0,0 +1,57 @@
+    try:
+        if sys.platform == "win32":
+            webview.start(debug=debug, gui="edgechromium")
+        else:
+            webview.start(debug=debug)
+    except Exception as exc:  # noqa: BLE001 - desktop startup should explain missing WebView2 clearly.
+        if sys.platform == "win32":
+            raise RuntimeError(
+                "PlexMuxy GUI requires Microsoft Edge WebView2 Runtime.\n\n"
+                "Windows 11 usually includes it. Some Windows 10 devices may need a manual install.\n\n"
+                f"Download: {WEBVIEW2_DOWNLOAD_URL}"
+            ) from exc
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)